lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
Well, then `__all__` would never be needed in python, right?

On Wed, Jan 6, 2016 at 1:53 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/figure.py
> <https://github.com/matplotlib/matplotlib/pull/5718#discussion_r48993652>:
>
> > @@ -32,7 +32,7 @@
> >
> >  from matplotlib.cbook import Stack, iterable
> >
> > -from matplotlib import _image
> > +from matplotlib import image as mimage
>
> eh, people should not be bulk importing from figure....right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5718/files#r48993652>.
>